docs/api/carts.rst

```diff
@@ -0,0 +1,764 @@
+Carts API
+=========
+
+These endpoints will allow you to easily manage cart and cart items. Base URI is `/api/v1/carts/`.
+
+.. note::
+
+    If you still don't know the difference between cart an order in Sylius context please read `this article`__ carefully
+
+__ http://docs.sylius.org/en/latest/book/index.html#orders
+
+If you request a cart, you will receive an object with following fields:
```

Comment:
I would add chapters here.

Like `Cart API response structure`, `CartItem API response structure`... and so on.